tests/executors/test_dask_executor.py

```diff
@@ -20,31 +20,39 @@
 from unittest import mock
 
 import pytest
+from distributed import LocalCluster
 
 from airflow.exceptions import AirflowException
+from airflow.executors.dask_executor import DaskExecutor
 from airflow.jobs.backfill_job import BackfillJob
 from airflow.models import DagBag
 from airflow.utils import timezone
 from tests.test_utils.config import conf_vars
 
 try:
-    from distributed import LocalCluster
-
     # utility functions imported from the dask testing suite to instantiate a test
     # cluster for tls tests
+    from distributed import tests  # noqa
     from distributed.utils_test import cluster as dask_testing_cluster, get_cert, tls_security
 
-    from airflow.executors.dask_executor import DaskExecutor
-
     skip_tls_tests = False
 except ImportError:
     skip_tls_tests = True
+    # In case the tests are skipped because of lacking test harness, get_cert should be
+    # overridden to avoid get_cert failing during test discovery as get_cert is used
+    # in conf_vars decorator
+    get_cert = lambda x: x
 
 DEFAULT_DATE = timezone.datetime(2017, 1, 1)
 SUCCESS_COMMAND = ['airflow', 'tasks', 'run', '--help']
 FAIL_COMMAND = ['airflow', 'tasks', 'run', 'false']
 
+# For now we are temporarily removing Dask support until we get Dask Team help us in making the
+# tests pass again
+skip_dask_tests = True
+
 
+@pytest.mark.skipif(skip_dask_tests, reason="The tests are skipped because it needs testing from Dask team")
```

Comment:
```suggestion
@pytest.mark.skip(reason="The tests are skipped because it needs testing from Dask team")
```
`skipif` is used to dynamically skip tests depending on a dynamic value, e.g. in the active database backend.

Reply:
I wanted to give an easy way for Dask committers to be able to "play" with it.  it's easier to set up one variable to True to do it, rather than manually remove @pytest.mark.skip

Reply:
I imagine the Dask developers will do it this way:

1) setup Breeze
2) get Dask service to run tests on
3) set `skip_dag_tests` to False
4) fix the tests
5) remove skips

By having single flag to switch that enables all tests it is just easier to not forget about removing some of the skips.